assets/kube-state-metrics/deployment.yaml

```diff
@@ -57,7 +57,22 @@ spec:
           ^kube_pod_completion_time$,
           ^kube_pod_status_scheduled$
         image: registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.12.0
+        livenessProbe:
+          httpGet:
+            path: livez
+            port: 8443
+            scheme: HTTPS
```

Comment:
(point `livenessProbe` for the KSM container to it's respective proxied port, this helps us actually restart the container on a probe failure)